src/sync/outgoingSyncUtils.js

```diff
@@ -188,21 +185,18 @@ const generateSyncData = (settings, recordType, record) => {
         status: STATUSES.translate(record.status, INTERNAL_TO_EXTERNAL),
         mode: record.mode,
         prescriber_ID: record.prescriber && record.prescriber.id,
-        total,
-        foreign_currency_total: total,
+        total: String(record.total),
+        foreign_currency_total: String(record.total),
         their_ref: record.theirRef,
         confirm_date: getDateString(record.confirmDate),
-        subtotal: total,
+        subtotal: String(record.total),
```

Comment:
Nite: might be nicer to do something like:

```
const total = String(record.total);
return {
  ...
  total,
  foreign_currency_total: total,
  ...
  subtotal: total.
  ...
};
```

Or, something I just found out works in JS:


```
const foreign_currency_total = subtotal = total = String(record.total);
return {
  ...
  total,
  foreign_currency_total,
  ...
  subtotal,
  ...
};
```

😆 .

Reply:
about to do a new PR for #2134 that changes more things regarding total, so will change this up there